### DISCUSSION TITLE
Time format 24h

### DISCUSSION THREAD

**Discussion by hari-bo** [marcokreeft87/formulaone-card#124] [Q&A] [answered]
I want to use 24h time notification instead of 12 hour am/pm.
Which property do i need to use? I've tried the following;
time_locale: nl
DateTimeFormat: nl

**Comment by marcokreeft87** [comment, marked as answer]
It uses the formatting of your HA by default. The only configurable option the card has is date_locale but that should only affect the dates.

**Reply by hari-bo** [reply]
Too bad, because i like the admin interface in english but the end user is using dutch language.
A workaround for me create a seperate user only for the end user and change the langua within the user profile and keep admin english.

**Reply by marcokreeft87** [reply]
Why go through all that trouble? 

![image](https://user-images.githubusercontent.com/10223677/215790943-93d77962-a5fb-4ca6-a386-234e9e4b49de.png)

**Reply by marcokreeft87** [reply]
You can set the language and time format there

**Reply by hari-bo** [reply]
> You can set the language and time format there

Just as i descrided, i only want to use English for the admin user that's why i use a seperate user on my mobile devices for the dasboard in Dutch language.

And not changing the language every x time, set and forget. :)

**Reply by marcokreeft87** [reply]
But do you want the time format in AM/PM for the admin user?

**Reply by hari-bo** [reply]
> But do you want the time format in AM/PM for the admin user?

Yes for admin purpose is English my prefered languague and won't use the user for accessing dashboard with mobile devices anymore. And all the users are Dutch.
